cmd/gateway_server/server/main.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+
+	"github.com/WICG/webpackage/go/signedexchange"
+	"github.com/WICG/webpackage/go/signedexchange/certurl"
+	pb "github.com/amaltas/gateway"
+	"github.com/ampproject/amppackager/packager/rtv"
+	"github.com/ampproject/amppackager/packager/signer"
+	"github.com/ampproject/amppackager/packager/util"
+	// For this import to work, delete vendor/julienschmidt from
+	// amppackager project. I couldn't figure out how to fix this.
```

Comment:
As you told me in person, this comment is no longer true. I was able to build without doing this.